src/dune_rules/super_context.mli

```diff
@@ -73,7 +73,8 @@ val odoc : t -> dir:Path.Build.t -> Env_node.Odoc.t Memo.t
 val coq : t -> dir:Path.Build.t -> Env_node.Coq.t Action_builder.t Memo.t
 
 (** Formatting settings in the corresponding [(env)] stanza. *)
-val format_config : t -> dir:Path.Build.t -> Format_config.t Memo.t
+val format_config :
+  t -> dir:Path.Build.t -> Ordered_set_lang.t Format_config.Generic.t Memo.t
```

Comment:
it would be nice to make an alias for `Ordered_set_lang.t Format_config.Generic.t` which appears in many places of `dune_rules` but I'm not sure what's the best place to put it.

Reply:
Maybe in a new file in dune rules ?

Reply:
let's scratch that idea for now